lib/ridgepole/default_limit.rb

```diff
@@ -23,7 +23,11 @@ def default_limit(column_type, options)
       end
 
       def adapter
-        ActiveRecord::Base.connection_config.fetch(:adapter).to_sym
+        if ActiveRecord.gem_version >= Gem::Version.new('6.1.0')
+          ActiveRecord::Base.connection_db_config.configuration_hash.fetch(:adapter).to_sym
+        else
+          ActiveRecord::Base.connection_config.fetch(:adapter).to_sym
+        end
```

Comment:
Fixed in https://github.com/winebarrel/ridgepole/pull/328. Would you please rebase? 🙏